registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -90,6 +90,7 @@
     "sha256:f9b6e51a8f5a6fc68dc02d8bd3d2aca39305823bf4c670689bcd835a4035612a": ["v1.10.2"]
     "sha256:d09af4d8cbb6935a13123732b5970c9e5ac3b12a9595493e43f65bf95ea8e9ba": ["v1.10.3"]
     "sha256:bf8b7c3ed427086138e673268edc720407bd8cb7caf2e2aa3a0e3d68277740e5": ["v1.10.4"]
+    "sha256:900b1cb008192a4bb19af9e5c7ebc0cff31dcb2c4b0b4cbb656da294c49d1fdd": ["v1.11.0"]
```

Comment:
SHA `900b1cb00819` matches cluster-api-controller v1.11.0 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/cluster-api-controller@sha256:900b1cb008192a4bb19af9e5c7ebc0cff31dcb2c4b0b4cbb656da294c49d1fdd/details?inv=1&invt=Ab56Hw).